app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_clss: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_clss
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self, distance: float, power: float, rating: float, count: int
+    ) -> None:
+        self.distance_from_city_center = distance
+        self.clean_power = power
+        self.average_rating = rating
+        self.count_of_ratings = count
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(car.comfort_class * (self.clean_power - car.clean_mark)
+                     * self.average_rating / self.distance_from_city_center, 1)
```

Comment:
can you divide formula into smaller chunks?

Reply:
Not sure if I got it right, tried to following "good practice"